ansible/playbooks/roles/compatibility_layer/tasks/install_prefix.yml

```diff
@@ -171,7 +219,10 @@
     - build_prefix
 
 - name: "Continue Gentoo Prefix bootstrap via {{ prefix_install }}"
-  ansible.builtin.shell: "set -o pipefail && {{ prefix_install }} | tee -a {{ prefix_build_log }} | grep -E '^(>>> Installing|\\* )'"
+  ansible.builtin.shell: set -o pipefail && ( {{ prefix_install }} | tee -a {{ prefix_build_log }} | grep -E '^(>>> Installing|\\* )' )
+  args:
+    executable: /bin/bash
+  become: false
```

Comment:
Can be removed.

Reply:
Hmm, for me `-o pipefail` was failing when `sh` was not symlinked to `bash` I think. Therefore, solution seemed to be to set the executable explicitly.

Reply:
Oh sorry, selected all lines here, only meant the last one `become: false` that shouldn't be necessary (we run everything as a regular user anyway in the container. The `bash` thing itself should definitely stay, indeed.

Reply:
`become...` removed with https://github.com/EESSI/compatibility-layer/pull/177/commits/ca44f10c816f217c1bd49db6e40f0e02837b9453